source/server/services/user-creation-service.js

```diff
@@ -9,16 +9,35 @@ Space.Object.extend(Space.accounts, 'UserCreationService', {
     Space.messaging.EventPublishing
   ],
 
-  statics: {
-    handleUserCreation(options, user) {
-      // Assign guids as user ids
-      user._id = options.userId;
+  onDependenciesReady() {
```

Comment:
@DominikGuzei, @rhyslbw: I have reverted back to static handler for Accounts.onCreateUser. It really fails in tests because of this:

https://github.com/meteor/meteor/blob/devel/packages/accounts-base/accounts_server.js#L118

This is a problem because we can not send an event that user account has been created from the static block when user was created implicitly after users has logged in for the first time using one of the OAuth services. 

Explicit user creation (initiated by command when creating user with user-password) is OK:
https://github.com/meteor-space/accounts/blob/feature/login-with-google/source/server/services/user-creation-service.js#L42

And we need this event on the server side, especially for ES applications.

Accounts-ui package is not using any of the events from the accounts package, it is sending it's own events on the client side only:
https://github.com/meteor-space/accounts-ui/blob/develop/source/client/controllers/login-controller.js#l17-l32

Any ideas how to solve this?